src/components/avatar/avatar.material.scss

```diff
@@ -0,0 +1,74 @@
+@use '../../styles/common/component';
+@use '../../styles/utilities' as *;
+
+:host {
+  display: block;
```

Comment:
This should rather be `inline-block` than `block` as we would want the element to appear alongside other comopnents/elements.